v1/openapi/parameters/filtering/timestamp_lte.yaml

```diff
@@ -1,7 +1,7 @@
 x-component: 'parameters/timestamp_lte'
 in: query
 name: timestamp_lte
-description: Ending point in a range. Greater than or equal to this timestamp in Unix format.
+description: Ending point in a range. Greater than or equal to this `timestamp` in Unix format.
```

Comment:
Less than

Reply:
already fixed in my last non-committed commit